### ISSUE TITLE
Wrong irregular pluralization with namespaced model name

### ISSUE THREAD

**Issue by artych** [stefanpenner/ember-inflector#77]

**Comment by stefanpenner** [comment]
i think this issue was valid, why is it closed?

**Comment by artych** [comment]
I thought it is more Ember.String issue
```javascript
Ember.String.pluralize('parties/person'); 
//=> 'parties/persons' (expected 'parties/people')
```

but yes it is valid for Inflector too
```javascript
Ember.Inflector.inflector.pluralize("parties/person");
//=> 'parties/persons' (expected 'parties/people')
```

**Comment by stefanpenner** [comment]
Yup this is a bug

**Comment by ofbriggs** [comment]
I've submitted a PR that should fix this: #78
